debezium-core/src/main/java/io/debezium/connector/base/ChangeEventQueueMetrics.java

```diff
@@ -5,6 +5,10 @@
  */
 package io.debezium.connector.base;
 
+/**
+ * @deprecated Will be replaced by {@link io.debezium.pipeline.metrics.todo.QueueMetricsMXBean}.
+ */
+@Deprecated
```

Comment:
No need to deprecate, let's just remove it and adjust our other repos accordingly. The general stance is:

* We care about compatibility for anything _user_-facing; i.e. config options, change event formats, metrics, embedded engine APIs, SPIs, etc.
* We don't care about compatibility in framework parts for now, as we control (almost) all implementations ourselves and can simply adjust them as needed; the only exception is the ScyllaDB connector which is maintained externally; for them, such change will be disruptive, but I think it's still manageable